### DISCUSSION TITLE
[VUE/NUXT] How to overwrite theme colors in layout?

### DISCUSSION THREAD

**Discussion by mateoKutnjak** [saadeghi/daisyui#2386] [Q&A]
I am doing a web app with multiple showcase web apps inside and I want for every app to have different primary color. How can I overwrite this variable (--p) in some path and all its children pages? Thanks

**Comment by saadeghi** [comment]
You can define each theme in `tailwind.config.js`:  
https://daisyui.com/docs/themes/#-4

And use them by adding `data-theme="THEME_NAME"` to `<html>` tag (or any other tag - of you want the theme only for a specific section of the page)

You can also define the color CSS variables manually. This way:  
https://unpkg.com/browse/daisyui@3.8.2/dist/themes.css

If you have a `/page/` and `/page/subpage/` you should use the `data-theme` or the CSS variables in the parent layout. It depends on your framework how to have a layout page for a directory and all its subdirectories.

Let me know if you have any questions.
